src/features/session_trace/instrument/index.js

```diff
@@ -18,7 +20,11 @@ export class Instrument extends InstrumentBase {
   static featureName = FEATURE_NAME
   constructor (agentIdentifier, aggregator, auto = true) {
     super(agentIdentifier, aggregator, FEATURE_NAME, auto)
-    if (!isBrowserScope) return // session traces not supported outside web env
+    const canTrackSession = isBrowserScope && getConfigurationValue(agentIdentifier, 'privacy.cookies_enabled') === true
```

Comment:
This is the exact check done by existing `enableSessionTracking`. We shouldn't duplicate it if there's really no reason to; can you use the existing fn?